### ISSUE TITLE
A few suggestions

### ISSUE THREAD

**Issue by chinakook** [ilkarman/DeepLearningFrameworks#20]
1. The last line can be changed to:
'print("Accuracy: ", sum(y_guess == y_test)/float(len(y_guess)))'
as I've got 0 in Python 2.7
2. Increace EPOCHES to 20 or more to get a more fair accuracy because of the differences in platform(such as hyper-parameters) and weights initialization and randomness with SGD.
3. Is the data need to zero mean normalization first to get a more resaonable result?

**Comment by ilkarman** [comment]
Regarding 3. instead of doing the equivalent of tf.image.per_image_standardization ((x - mean) / adjusted_stddev), the pixel intensities are just re-scaled. This probably degrades performance but shouldn't affect the relative difference between frameworks. I will play around to see if it makes a big difference.

**Comment by botev** [comment]
Also, actually it might be worth to have a few iterations of "burn-in" before you time them, especially with the cuDNN auto-tune some of the frameworks will run the procedure when you build the computations while other will do it on the first call to any execution. Thus it is unfair to the latter if you just time them. I would suggest no more than 10 iterations for such burn-in phase, this should give a very tiny overhead on the whole execution.

**Comment by kirnap** [comment]
I have a question regarding your [RNN tensorflow](https://github.com/ilkarman/DeepLearningFrameworks/blob/master/Tensorflow_RNN.ipynb) implementation. Since you padded sequences with zero,  don't you need to implement some kind of masking mechanism, so that the loss contribution coming from pad character neutralized? How do you handle with that without implementing mask matrix? Does Tensorflow do it automatically?
